app/helpers/model_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ModelHelpers
```

Comment:
your helper modules are intended for your views - can we find another place to put these methods?